core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1855,6 +1857,28 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Because we can only have one active Spark Context per JVM and there are times when multiple
+   * applications may wish to share a SparkContext, this function may be used to get or instantiate
+   * a SparkContext and register it as a singleton object.
+   * 
+   * Note: This function cannot be used to create multiple spark contexts even if multiple contexts
+   * are allowed. Multiple contexts will still need to be created with explicit calls to the 
+   * SparkContext constructor. 
```

Comment:
I would just leave out this line because we actually discourage users from creating multiple SparkContexts in the same JVM